genai-perf/genai_perf/export_data/console_exporter.py

```diff
@@ -84,6 +79,19 @@ def _construct_table(self, table: Table) -> None:
 
             table.add_row(*row_values)
 
+        for metric in self._metrics.system_metrics:
+            metric_str = metric.name.replace("_", " ").capitalize()
+            metric_str = metric_str.replace("throughput", "tput")
```

Comment:
why are we replacing throughput to tput?

Reply:
The console gets too wide otherwise and risks cutting off the data.
The data seemed more important than the non abbreviated metric text.

Reply:
I agree, but I feel like this needs a separate discussion because then we're losing consistency in our visual display. Since console is solely for visual look, and people who need the data can look for csv or json, could we keep the full name for consistency for now? If this becomes a problem, then we can bring the discussion to the table along with other metrics as well.

Reply:
I am confused by how we are losing consistency in our visual display?
This is the only place where it is displayed.

Reply:
what I mean is no other metrics are abbreviated except the two added metrics.

Reply:
I see. At the moment, we have sign off from product who I think would be the final call on this. Still happy to discuss offline if that is not sufficient for your concerns.

Reply:
I'm okay with change if product decides to do so. Just wanted to discuss about this before we change anything on screen that users see directly.

Could you update the screenshot above with the `tput` output? 

cc @harryskim for final call

Reply:
Updated the screenshot

Reply:
reverted the abbreviation. should be all set now